roles/cdi/templates/cdi-controller.yaml.j2

```diff
@@ -0,0 +1,322 @@
+kind: ServiceAccount
```

Comment:
why do you introduce this? is it related to the patch?
I have a vague memory that we ship (and want to keep shipping) the cdi manifest elsewhere, which kubevirt-ansible.rpm requires.

Reply:
I need to check I think the cdi download it and save it there in the playbook run time. I didn't add this file manually